instrumentation/manticore/lib/opentelemetry/instrumentation/manticore/patches/response.rb

```diff
@@ -0,0 +1,104 @@
+          def request_attributes(wrapped_request)
+            uri = URI.parse(wrapped_request.uri)
+            attr = {
+              'library' => 'Manticore',
+              'http.method' => wrapped_request.method,
+              'http.scheme' => uri.scheme,
+              'http.target' => uri.path,
+              'http.url' => "#{uri.scheme}://#{uri.host}",
+              'net.peer.name' => uri.host,
+              'net.peer.port' => uri.port
+            }
+            return attr if Manticore::Instrumentation.instance.config["record_request_headers_list"].empty?
+            header_attr = header_attributes(wrapped_request.headers,
+                                            Manticore::Instrumentation.instance.config["record_request_headers_list"],
+                                            'http.request')
+            attr.merge(header_attr)
+          end
+
+          # @param [Manticore::Response] response A @response instance object after an outgoing call is attempted
+          # @return [Hash] returns response attributes to be added to span.
+          def response_attributes(response)
+            attr = {
+              'http.status_code' => response.code,
+              'http.status_text' => response.message
+            }
+            return attr if Manticore::Instrumentation.instance.config["record_response_headers_list"].empty?
+            header_attr = header_attributes(response.headers,
+                                            Manticore::Instrumentation.instance.config["record_response_headers_list"],
+                                            'http.response')
+            attr.merge(header_attr)
+          end
+
+          # @param [OpenTelemetry::Instrumentation::Manticore::Util::WrappedRequest.headers] headers receives the headers hash of the manticore request object
+          # @param [Array] record_headers_list an array that should include a list of interested headers that would like to be recorded. The headers provided should be from either 'record_response_headers_list' or 'record_request_headers_list' configured when starting the instrumentation installation.
+          # @param [String] request_type a string that prefix of all the keys to be recorded in the attribute. The string should only be 'http.request' or 'http.response' This is specified in
+          # `https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#http-request-and-response-headers`.
+          # @return [Hash] A duplicated headers attributes hash to be added to the span
+          def header_attributes(headers, record_headers_list = [], request_type = 'http.request')
+            return {} if headers.empty? || record_headers_list.empty?
+            attr = {}
+            record_list = headers.keys & record_headers_list
+            record_list.each do |key|
+              attr["#{request_type}.#{key}"] = headers[key]
```

Comment:
The format of the attribute should be `http.request.headers.<key>` or `http.response.headers.<key>` according to the spec (it may have been updated recently, sorry again for the delay in reviewing here)

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#http-request-and-response-headers
```suggestion
              attr["#{request_type}.headers.#{key}"] = headers[key]
```